{"version":1,"defects":{"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #2":4,"Error":4,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #0":4,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #1":4,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #3":4,"Tests\\Unit\\RouterTest::it_test_from_a_closure":4,"Tests\\Unit\\ContainerTest::it_checks_not_callable_case_for_get_without_setting":4,"Tests\\Unit\\Controllers\\FlightEditorControllerTest::it_checks_exception_should_return_array_of_airplanes":5,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::redirect_to_bad_request_page":3,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::redirects_to_all_flights_page":3,"Tests\\Unit\\Models\\FindAllFlights::checks_example_result_with_fetch with data set #0":3,"Tests\\Unit\\Models\\FindAllFlights::checks_warnings_with_fetch with data set #0":3,"Tests\\Unit\\Models\\FindAllFlights::checks_warnings_with_fetch with data set #1":3,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::shows_incorrect_password_message":3},"times":{"Tests\\Unit\\RouterTest::it_registers_a_route":0.005,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #0":0.004,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #1":0,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #2":0,"Tests\\Unit\\RouterTest::it_throws_unknown_uri_esception with data set #3":0,"Error":0.082,"Tests\\Unit\\RouterTest::it_test_from_a_closure":0.096,"Tests\\Unit\\RouterTest::it_tests_resolution_a_route":0,"Tests\\Unit\\ContainerTest::it_checks_has_function":0.115,"Tests\\Unit\\ContainerTest::it_checks_callable_case":0,"Tests\\Unit\\ContainerTest::it_checks_not_callable_case_for_get":0.023,"Tests\\Unit\\ContainerTest::it_checks_not_callable_case_for_get_without_setting":0,"Tests\\Unit\\ContainerTest::it_checks_not_callable_case_for_get_with_setting":0.02,"Tests\\Unit\\ContainerTest::it_checks_resolve_for_uninstantiable_class":0.056,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_has_function":0.09,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_callable_case":0,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_not_callable_case_for_get_with_setting":0.027,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_not_callable_case_for_get_without_setting":0,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_resolve_for_uninstantiable_class":0.036,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_resolve_without_constructor":0.007,"Tests\\Unit\\ContainerTest\\ContainerTest::it_checks_resolve_with_constructor":0.008,"Tests\\Unit\\Controllers\\FlightEditorControllerTest::it_checks_exception_should_return_array_of_airplanes":0.009,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::shows_incorrect_login_message":0.321,"Tests\\Unit\\RouterTest::it_tests_resolution_from_a_closure":0,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::shows_incorrect_password_message":0.016,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::redirect_to_bad_request_page":0,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::redirects_to_all_flights_page":0.011,"Tests\\Unit\\Controllers\\PlannerLoginControllerTest::redirect_to_all_flights_if_logged":0,"Tests\\Unit\\Models\\FindAllFlights::checks_example_result_with_fetch with data set #0":0.025,"Tests\\Unit\\Models\\FindAllFlights::checks_warnings_with_fetch with data set #0":0.411,"Tests\\Unit\\Models\\FindAllFlights::checks_warnings_with_fetch with data set #1":0}}